src/app/app.ts

```diff
@@ -150,6 +150,47 @@ export function view(state: IState, actions: any) {
       id: 'app',
       class: classNames({'theme-dark': state.settings.darkTheme}),
       oncreate: (el: El) => actions.onCreate(el),
+      ondragover: (ev: Event) => {
+        ev.preventDefault()
+        ev.stopPropagation()
+      },
+      ondragleave: (ev: Event) => {
+        ev.preventDefault()
+        ev.stopPropagation()
+      },
+      ondrop: (ev: any, node: IGraphNodeData) => {
+        ev.preventDefault()
+        ev.stopPropagation()
+        for (const f of ev.dataTransfer.files) {
+          logger.debug('Dropped file path = ' + f.path)
+          console.log(f.path)
+          if (f.path.includes('.txt')){
+            const reader = new FileReader()
+            reader.readAsDataURL(f)
+            reader.onloadend = () => {
+              const base64Data: string = reader.result!.toString().split(',')[1]
+              const fileContent = atob(base64Data)
+              actions.graph.createNewNode({
+                position: {x: ev.screenX, y: ev.screenY},
+                parent: null,
+                newNodeCallback: (nodeId: GraphNodeId) => {
+                  writeNote(nodeId, NoteDataType.TEXT, fileContent)
+                },
```

Comment:
did you fix the note type specification issue as we discussed? I dont see the changes in the graph actions and the last argument to `createNewNode` specifying the note type as text or handwriting.

Reply:
I fixed that, I'm putting in the image part. Will push again later